numpy/testing/_private/utils.py

```diff
@@ -386,7 +386,12 @@ def assert_equal(actual, desired, err_msg='', verbose=True):
             if not signbit(desired) == signbit(actual):
                 raise AssertionError(msg)
 
-    except (TypeError, ValueError, NotImplementedError):
+    except (TypeError, ValueError, NotImplementedError, DeprecationWarning):
+        # version 1.18:
+        # datetime64/timedelta64 used to raise since they had no isnan
```

Comment:
To be clear - `gisnan` would raise, or `signbit` would raise?

Reply:
`gisnan` used to raise and no longer does. `signbit` still raises. I will try to clarify